docs/compute.md

```diff
@@ -344,7 +344,24 @@ Other useful commands for listing the VM core affinity and cores per NUMA node(s
 ```
 xl vcpu-list
 xenpm get-cpu-topology
+xl info --numa
 ```
+
+Other ways control the vCPUs placement, using the `xl` CLI:
+```
+xl vcpu-pin <Domain> <vcpu id> <cpu id>
+xl vcpu-pin Domain\ 0 all 2-5
```

Comment:
Maybe prefer using `"` rather than `\`?

Reply:
And also where do you get the exact name of the domain from?

Reply:
Okay, it's the name from `xl list`, it might work with the ID too.